app/views/generic_files/_generic_file.html.erb

```diff
@@ -1,6 +1,7 @@
 <li>
   <h2>
     <%= link_to generic_file.title_or_label, sufia.generic_file_path(generic_file) %>
+    <small>[<%= link_to 'Download', download_path(generic_file), target: '_blank', rel: 'noopener noreferrer' %>]</small>
```

Comment:
Yeah, that's the conclusion I'm coming to. I'll need to refactor the helper to not assumer it's working with a generic file, since when this partial is called on search and collection pages, it's working with solr docs